src/h5web/vis-packs/core/shared/PanZoomMesh.tsx

```diff
@@ -93,6 +93,7 @@ function PanZoomMesh(): ReactElement {
 
   return (
     <mesh {...{ onPointerMove, onPointerUp, onPointerDown, onWheel }}>
+      <meshBasicMaterial attach="material" color="#000000" opacity={0} />
```

Comment:
I know the color is not visible, that's the point, but maybe use `#ffffff` for clarity?

Reply:
> maybe use `#ffffff` for clarity?

I see what you did there :smirk: